api/agent/drivers/docker/docker_test.go

```diff
@@ -574,7 +574,7 @@ func TestRunnerDockerInvalidSyslog(t *testing.T) {
 	if err == nil {
 		t.Fatal("Error expected when running with invalid syslog configuration")
 	}
-	if err.Error() != "Syslog Unavailable" {
+	if err.Error() != models.ErrSyslogUnavailable.Error() {
```

Comment:
Can this not just be `if err != models.ErrSyslogUnavailable`?

Reply:
Done